docs/installation.rst

```diff
@@ -32,8 +38,42 @@ Or you can start a python interpreter and try importing SpiceyPy like so:
 This should print out the toolkit version without any errors. You have now
 verified that SpiceyPy is installed.
 
-How to install from source (for bleeding edge)
-----------------------------------------------
+=============
+Common Issues
+=============
+
+SSL Alert Handshake Issue
+-------------------------
+In early 2017, JPL updated to a TLS1.2 certificate and enforced https connections causing installation
+issues for users, in particular for macOS users, with OpenSSL versions older
+than 1.0.1g. This is because older versions of OpenSSL still distributed in some
+environments which are incompatible with TLS1.2. As of late 2017 SpiceyPy has been updated with
+a strategy that can mitigate this issue on some systems, but it may not be totally reliable due to known deficiencies in setuptools and pip.
+
+Another solution is to configure a new python installation that is linked against a newer version
+of OpenSSL, the easiest way to do this is to install python using homebrew, once this is done spiceypy
+can be installed to this new installation of python (IMHO this is the best option).
+
+If your python distribution was installed from the packages available at python.org an included command
```

Comment:
This command is only distributed for Python 3.6 (and higher).  For earlier versions, the only way out is the home-brew installation.